concordium-consensus/src/Concordium/MultiVersion.hs

```diff
@@ -729,7 +750,8 @@ startupSkov genesis = do
                         case nextPV of
                             Nothing -> do
                                 mvrLogIO $ activateConfiguration newEConfig
-                                liftSkovUpdate newEConfig' checkForProtocolUpdate
+                                _ <- liftSkovUpdate newEConfig' checkForProtocolUpdate
```

Comment:
This is not exactly ideal since if the user has shut down the node and then restarted it again the flag will say 0 even though the node is in a state where it has an unrecognized protocol update.

Reply:
I have taken a different approach now, by introducing a dedicated callback just for signalling a pending unsupported protocol update.